ydb/core/protos/table_stats.proto

```diff
@@ -55,4 +55,11 @@ message TTableStats {
     optional bool HasLoanedParts = 29;
 
     repeated TChannelStats Channels = 30;
+
+    message TStoragePoolStats {
+        optional string PoolKind = 1;
+        optional uint64 DataSize = 2;
+        optional uint64 IndexSize = 3;
+    }
+    repeated TStoragePoolStats StoragePools = 31;
```

Comment:
TO DO: add a comment stating that this field was added solely for the informative PathDescription in the UI.